kong/plugins/zipkin/opentracing.lua

```diff
@@ -48,23 +46,21 @@ end
 
 function OpenTracingHandler:initialise_request(conf, ctx)
 	local tracer = self:get_tracer(conf)
-	local var = ngx.var
-	local wire_context = tracer:extract("http_headers", ngx.req.get_headers()) -- could be nil
-	local method, request_url -- if reading request line fails then var.request_uri is nil
-	if var.request_uri then
-		method = ngx.req.get_method()
-		request_url = var.scheme .. "://" .. var.host .. ":" .. var.server_port .. var.request_uri
-	end
+	local req = kong.request
+	local wire_context = tracer:extract("http_headers", req.get_headers()) -- could be nil
+	local url = req.get_scheme() .. "://" .. req.get_host() .. ":"
+	            .. req.get_port() .. req.get_path_with_query()
+	local forwarded_ip = kong.client.get_forwarded_ip()
```

Comment:
I don't think we want to respect `Forwarded-For` here. Does `forwarded_ip` do that?

Reply:
`kong.client.get_forwarded_ip()` is just a very thin wrapper around `ngx.var.remote_addr`, which is what the old code did:

https://github.com/Kong/kong/blob/master/kong/pdk/client.lua#L76

Reply:
Okay then, it looks like the old behaviour is retained, I didn't realise the ngx_real_ip module overwrites `remote_addr` with the fake ip.